### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by angrycaptain19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,78 @@
 
 
 ## Sourcery
-Refactor Python instantly and keep your codebase maintainable.
+Instantly review and improve your Python code to keep your codebase maintainable. 
 
-## GitHub Code Reviews
+Get started with `pip install sourcery-cli` to have Sourcery review all your code locally or star this repo to have our GitHub bot review your most popular Python repo
```
Comment (Hellebore) [inline README.md:7]: do we want this duplication of the pip install with the section just below?

File: README.md [hunk 2]
```diff
@@ -2,32 +2,78 @@
...
+
+## CLI
+
+You can have Sourcery review your entire project and make dozens or hundreds (or thousands) of improvements with a single command from the command line. 
+
+To start `pip install sourcery-cli`, login with `sourcery login`, and then run `sourcery review <some path>` to have Sourcery review and make suggestions on every file in that path
+
+## CI & Pre-Commit
+
+Keep your entire codebase standardized with Sourcery’s CI integration and pre-commit hook.
```
Comment (Hellebore) [inline README.md:17]: we should also link to our CI docs

**Review by Hellebore — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (angrycaptain19) [reply README.md:7]: Good call - can cut

File: README.md (hunk 2 above)
Comment (angrycaptain19) [reply README.md:17]: 👍

**Pull request merged**